app/api/events.py

```diff
@@ -12,17 +16,61 @@
 
 
 class EventSchema(Schema):
-
     class Meta:
         type_ = 'event'
         self_view = 'v1.event_detail'
-        self_view_kwargs = {'id': '<id>'}
+        self_view_kwargs = {'id': '<id>', 'identifier': '<identifier>'}
         self_view_many = 'v1.event_list'
 
     id = fields.Str(dump_only=True)
     identifier = fields.Str(dump_only=True)
-    name = fields.Str()
-    event_url = fields.Str()
+    name = fields.Str(required=True)
+    email = fields.Str(required=True)
+    event_url = fields.Str(dump_only=True)
+    starts_at = fields.DateTime(required=True)
+    ends_at = fields.DateTime(required=True)
+    timezone = fields.Str()
+    latitude = fields.Float()
+    longitude = fields.Float()
+    logo_url = fields.Str()
```

Comment:
Check the response again, you won't be getting the `logo_url` field to rename logo to `logo_url`, you have to specify logo_url = fields.Str(attribute='logo'). Do similar for others.